parsers/flutter.go

```diff
@@ -18,13 +18,14 @@ type flutterTest struct {
 
 type flutterEntry struct {
 	Type   string
+	Error  string
 	TestID int64
 	Result string
 	Test   flutterTest
 }
 
-func annotationFromTest(test flutterTest) model.Annotation {
-	return model.Annotation{
+func annotationFromTest(test *flutterTest) *model.Annotation {
+	return &model.Annotation{
 		Type:    model.TestResultAnnotationType,
 		Level:   "failure",
```

Comment:
Let's not set level here then (since it's created on a test start). And instead set the level on testDone before appending the annotation.

Reply:
See https://github.com/cirruslabs/cirrus-ci-annotations/pull/31/commits/cf5507b6d061aa6ba9665d300d896825882ce00d.